validate_staging_area.py

```diff
@@ -4,33 +4,52 @@
 """
 import argparse
 import sys
+
 from hca.staging_area_validator import StagingAreaValidator
 
 
 def _parse_args(argv):
     parser = argparse.ArgumentParser(description=__doc__)
-    parser.add_argument('--staging-area', '-s',
-                        required=True,
-                        help='The Google Cloud Storage URL of the staging area. '
-                             'Syntax is gs://<bucket>[/<path>].')
-    parser.add_argument('--ignore-dangling-inputs', '-I',
-                        action='store_true',
-                        default=False,
-                        help='Ignore errors caused by metadata files not found '
-                             'in the staging area for input-only entities.')
-    parser.add_argument('--no-json-validation', '-J',
-                        action='store_false',
-                        default=True,
-                        dest='validate_json',
-                        help='Do not validate JSON documents against their schema.')
+    parser.add_argument(
+        "--staging-area",
+        "-s",
+        required=True,
+        help="The Google Cloud Storage URL of the staging area. "
+        "Syntax is gs://<bucket>[/<path>].",
+    )
+    parser.add_argument(
+        "--ignore-dangling-inputs",
+        "-I",
+        action="store_true",
+        default=False,
+        help="Ignore errors caused by metadata files not found "
+        "in the staging area for input-only entities.",
+    )
+    parser.add_argument(
+        "--no-json-validation",
+        "-J",
+        action="store_false",
+        default=True,
+        dest="validate_json",
+        help="Do not validate JSON documents against their schema.",
+    )
+    parser.add_argument(
+        "--total-retries",
+        "-t",
+        type=int,
+        default=10,
+        dest="total_retries",
```

Comment:
```suggestion
        dest="max_retries",
```


..and rename every `total_retries` variable / parameter to `max_retries`